libraries/Wire/src/Wire.cpp

```diff
@@ -250,7 +250,7 @@ size_t TwoWire::write(const uint8_t *data, size_t quantity)
             ret = 0;
         }
     }
-    return quantity;
+    return ret;
```

Comment:
If the function per API spec returns an error code, the DoxyGen comment above (`@retval ..`) must be adapted too.

Reply:
Oh right, I didn't see the above `size_t ret = quantity;` initially, thought `ret = 0;` above was an error / okay code. So this is okay.